esp/src/src-react/themes.css

```diff
@@ -17,15 +17,15 @@
     --neutralLighterAlt: "#faf9f8";
     --neutralLighter: "#f3f2f1";
     --neutralLight: "#edebe9";
-    --neutralQuaternaryAlt: "#e1dfdd";
```

Comment:
I had modified theme.ts a few weeks back, and just noticed the comment in this file about keeping it in sync